README.md

```diff
@@ -147,7 +147,7 @@ in `Cargo.toml`:
 
 ```toml
 [dependencies]
-derive_more = "0.99.0"
+derive_more = "1.0.0-beta.3"
```

Comment:
Let's for the beta suggest to pin the exact version, as breaking changes can happen in any beta version.
```suggestion
derive_more = "=1.0.0-beta.3"
```